pages/en/advanced/bp-sidecar.mdx

```diff
@@ -144,3 +144,15 @@ ERROR:root:Sleeping for 30s and trying again
 ```
 
 It likely means you're shipping off data to the ingest pipeline without any block producer key configured on your Mina node - since your BP key is your identity we can't accept node data since we don't know who is submitting it!
+
+
+### Relevant Links
+
+- [Latest Release Notes for Mina Protocol](https://github.com/MinaProtocol/mina/releases/tag/].3.0)
```

Comment:
`https://github.com/MinaProtocol/mina/releases/tag/].3.0` leads to a 404

Reply:
Fixed!